********************************************************************************
MAQAO 2.20.1 - cf6e61a781c49dfcbeaded3d28e83d3212ea2c5e::20240418-162820 || 2024/04/18
maqao OV -R1 external_libraries=my_functions.so xp=OV1_python -- /bin/python3 driver.py 
CPY:  [true] /bin/python3 --> /home/user/macao-kitchen/driver_python_c/OV1_python/binaries/python3
debuglink section found in /bin/python3 but target d0fd215a9a98d5333fda7e5a9a46c923148415.debug not found
CMD:   /usr/bin/maqao lprof  --xp="/home/user/macao-kitchen/driver_python_c/OV1_python/tools/lprof_npsu_run_0" --mpi-command=""  -ldi=my_functions.so  -- /home/user/macao-kitchen/driver_python_c/OV1_python/binaries/python3  driver.py
###   ERROR 1   ###
********************************************************************************
MAQAO 2.20.1 - cf6e61a781c49dfcbeaded3d28e83d3212ea2c5e::20240418-162820 || 2024/04/18
maqao OV -R1 external_libraries=my_functions.so xp=OV1_python -- /bin/python3 driver.py 
CMD:   /usr/bin/maqao lprof  --xp="/home/user/macao-kitchen/driver_python_c/OV1_python/tools/lprof_npsu_run_0" --mpi-command=""  -ldi=my_functions.so  -- /home/user/macao-kitchen/driver_python_c/OV1_python/binaries/python3  driver.py
###   ERROR 1   ###
